src/main.rs

```diff
@@ -80,89 +88,66 @@ fn run(root_dir: Option<PathBuf>, discard_failures: bool, scope: Option<&str>,
     Ok(invocation.exit_code())
 }
 
+#[derive(Parser)]
+#[clap(about, version)]
+struct Cli {
```

Comment:
I see you've reordered the fields vs. the order the flags were declared in ([command, stale, ttl, cache_dir, env, scope, cwd, warm, discard_failure, force] vs. [command, ttl, stale, warm, force, cwd, env, discard-failures, scope, cache_dir]) - any particular reason to change the ordering? I don't think it matters terribly much, but IMO ttl should come before stale, cwd and env should be next to each other, warm and force should be next to each other, etc. I'd vote to preserve the existing source ordering unless there's a reason you feel it's problematic.

Reply:
`clap` reorders args and flags alphabetically, I'll reorder to match the original and add `AppSettings::DeriveDisplayOrder` to force that

Reply:
Thanks, I'd actually just meant to keep the source order the same but looking at the help text with `DeriveDisplayOrder` I like that better than the default/existing ordering!